internal/dbutil/sqliteutil/db.go

```diff
@@ -14,7 +14,8 @@ import (
 
 // LoadSchemaFromSQL inserts script into connstr and loads schema.
 func LoadSchemaFromSQL(script string) (_ *dbschema.Schema, err error) {
-	db, err := sql.Open("sqlite3", ":memory:")
+	db, err := sql.Open("sqlite3", "file::memory:?mode=memory&_journal=WAL&_busy_timeout=30000")
```

Comment:
Let's make the constant public in `sqliteutil` and use it throughout the rest of the code. Or alternatively, let's create `sqliteutil.OpenInMemory()`.